sci-mathematics/cgal/cgal-4.10.2.recipe

```diff
@@ -0,0 +1,131 @@
+	1997-2004 Utrecht University (The Netherlands)
+	1997-2004 ETH Zurich (Switzerland)
+	1997-2004 Tel-Aviv University (Israel)"
+LICENSE="GNU GPL v3
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-$portVersion/CGAL-$portVersion.tar.xz"
+CHECKSUM_SHA256="5cc666e9813cb6515c1b4d14a167160be24d60c43c97fef8970dd3798a4c66a8"
+SOURCE_DIR="CGAL-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandSuffix=$secondaryArchSuffix
+	commandBinDir=$binDir
+else
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	cgal$secondaryArchSuffix = $portVersion
+	cmd:cgal_create_cmakelists
+	cmd:cgal_create_cmake_script
+	cmd:cgal_make_macosx_app
+	lib:libCGAL$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Core$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_ImageIO$secondaryArchSuffix = 13.0.0 compat >= 13
+	lib:libCGAL_Qt5$secondaryArchSuffix = 13.0.0 compat >= 13
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_atomic$secondaryArchSuffix
+	lib:libboost_chrono$secondaryArchSuffix
+	lib:libboost_date_time$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_thread$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libmpfr$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5OpenGL$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	cgal${secondaryArchSuffix}_devel = $portVersion
+	devel:libcgal$secondaryArchSuffix = 13.0.0 compat >= 13
+	devel:libCGAL_Core$secondaryArchSuffix = 13.0.0 compat >= 13
+	devel:libCGAL_ImageIO$secondaryArchSuffix = 13.0.0 compat >= 13
+	devel:libCGAL_Qt5$secondaryArchSuffix = 13.0.0 compat >= 13
+	"
+REQUIRES_devel="
+	cgal$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:eigen$secondaryArchSuffix
+	devel:libboost_atomic$secondaryArchSuffix
+	devel:libboost_chrono$secondaryArchSuffix
+	devel:libboost_date_time$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libboost_thread$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libmpfr$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5OpenGL$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DWITH_Eigen3=ON \
+		-DCGAL_INSTALL_CMAKE_DIR="lib/cmake"
```

Comment:
I think you mean:

```diff
-		-DCGAL_INSTALL_CMAKE_DIR="lib/cmake"
+		-DCGAL_INSTALL_CMAKE_DIR="$relativeLibDir/cmake"
```

Also, this project [appears to use `GNUInstallDirs`][0], so you might be able to avoid manually moving headers to the right place

[0]: https://github.com/CGAL/cgal/blob/releases/CGAL-4.10.2/Installation/CMakeLists.txt#L44